authors/apps/articles/models.py

```diff
@@ -43,3 +47,73 @@ class ArticleImage(models.Model):
 
     class Meta:
         ordering = ('-date_created',)
+
+class LikeArticles(models.Model):
+    user = models.ForeignKey(
+        User,
+        related_name='liked_by',
+        on_delete=models.CASCADE
+    )
+    article = models.ForeignKey(
+        Article,
+        to_field='slug',
+        related_name='liked',
+        db_column='article',
+        on_delete=models.CASCADE
+    )
+    likes = models.IntegerField(null=True)
+    dislikes = models.IntegerField(null=True)
+    created_on = models.DateTimeField(auto_now=True)
+    class Meta:
+        ordering = ('created_on',)
+
+    @staticmethod
+    def retrieve_article(slug):
```

Comment:
I have noticed you have created a new method to find an article by slug, I had initially created a method called find_article at the top of this file that you can reuse.